scopus/scopus_author.py

```diff
@@ -109,8 +109,7 @@ def __init__(self,
         level = number of * to print in __str__.
         """
 
-        if isinstance(author_id, int):
-            author_id = str(author_id)
+        author_id = str(author_id)
```

Comment:
Why do you think this is necessary? You could end up with 'None' as an author name in this case.